### PR TITLE
Feature/us5599 inline giving update prototype to match new mockups

### PR DESCRIPTION
**Pull request by zarahzachz**
*Update styles on Inline Giving*

- [x] Make changes to interface per revised designs here: https://invis.io/XU852E1HJ (exclude updates to amount steps at this time) - most of these are margin/padding/spacing updates as well as calendar style
- [ ] Update validation styles to match approved designs from Rob (https://www.dropbox.com/sh/esv6x2clkjet94j/AADyLkWZlPjg5cxtRtggNhL5a?dl=0)
- [x] embed needs to be 100% (with a max maybe) so it always fills the space it's placed within. See example http://codepen.io/bzmillerboy/full/vyNpVK/
- [ ] If we're going with a borderless input, I think the contrast to the background needs improved so the input is visible to those on monitors with lower contrast capabilities (non retina). *[Design - Rob's input]*
- [ ] input Don't disable next button until a value is picked. Validate required fields on click of next. *[Technical Req]*
- [x] amount input need to be bigger, see the padding in the design [Design - Rob's input (George's mockups don't have consistent spacing)]
- [x] Button heights are a bit short compared to design (particularly on mobile) [Design - ]
- [x] $ doesn't need to be bold.
- [x] Amount input doesn't bring up the number key pad (it did at one point, not sure how that was lost).
- [x] Inputs have border radius on them and in the design they are no border radius. This may help solve #4 above. *Didn't do, waiting for Rob's input →* Perhaps a bottom border would help (just idea). [US5599]
- [x] The base font size appears to be 14px. CR.net is currently set to 16px, I think it's probably best if we stick with that base font size for consistency.
- [x] The top and bottom of the content in embed will need some spacing so that when it's inserted as an iframe into another page, it's content doesn't sit directly up against another element. Just a little bit of room will keep it safe. Any fine tuning beyond that would be up to the developer inserting the iframe. https://www.dropbox.com/s/m48u7gt8517nlmr/Screenshot%202016-11-08%2014.03.33.png?dl=0
- [x] There is very little difference in the input text and text on the rest of the page, this makes it difficult to understand the input and placeholder text versus regular body text. The design seems to have more of a difference.
- [x] Is "register for an account" the right color? It looks darker than the design.
- [x] Space below all headers seems to be a bit more than the design. I like the tightness in the design.
- [x] Space between input elements is more spread out than in the design, I like the tightness in the design (particularly obvious on the registration page *← is this still a screen in the embed? taylor said registration might send people to a new window instead*).
- [x] I'm not even sure if "Account Type:" label is needed. If so, then we can make it a smaller heading (perhaps just a label) than the main "Payment Type" heading. [Design]
- [x] CC Expiration input should display a number pad just like CVV does because you're only ever entering numbers in that field.
- [x] On the summary page, the labels look really aggressive compared to the design. Maybe they don't need to be bold and could be listed as small labels.
- [x] Buttons should be outside the grey box, according to the design.
- [x] All back buttons should read "Back" not "Go Back"
- [ ] Disable zoom on page so the page doesn't zoom when users selects an input. (<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">) *Someone else did this?*

### CODE REVIEW

**Comment by beckysingson** [comment]
@zarahzachz 

 1. Odd spacing on the labels on the payment type screen:

crdsembed

**Comment by zarahzachz** [comment]
I'm having problems recreating this bug. I've tried in Chrome, Safari and the iPhone simulator. 

screen shot 2016-11-18 at 11 35 11 am

**Comment by beckysingson** [comment]
@zarahzachz just re-pulled everything, showing up fine now. I'll go ahead and green check this.

File: src/app/summary/summary.component.html [hunk 1]
```diff
@@ -1,44 +1,46 @@
...
-      <td><a (click)="changePayment()"><strong>Payment:</strong></a></td>
-      <td class="text-right">
-        Account ending in {{ lastFourOfAcctNumber | slice:-4 }}<br>
-        <small><a (click)="changePayment()">change</a></small>
-      </td>
-    </tr>
-  </table>
+<table class="table">
+  <tr>
+    <td *ngIf="gift.type == 'gift'"><a routerLink="/donation/amount">Amount:</a></td>
```
Comment (samueljmello) [inline src/app/summary/summary.component.html]: @zarahzachz All references to gift.type are either value "donation" or "payment". "gift" is not a valid value I think.

Comment (zarahzachz) [inline src/app/summary/summary.component.html]: I swapped out `'gift'` for `'donation'`

**Pull request merged**